Gemfile

```diff
@@ -1,11 +1,13 @@
 source 'https://rubygems.org'
 
-gem 'berkshelf',  '~> 3.2'
-gem 'chefspec',   '~> 4.2'
-gem 'foodcritic', '~> 4.0'
-gem 'rubocop',    '~> 0.30.1'
+gem 'berkshelf', '~> 4.3.5'
```

Comment:
If projects follow semver this works great since we can bring in 4.4, but not 5.0

gem 'berkshelf', '~> 4.3'